account_banking_nl_rabo_mt940/account_banking_nl_rabo_mt940.py

```diff
@@ -42,85 +39,57 @@ class RaboMT940Parser(MT940, parser):
         r'(?P<reference>\w{1,16})')
 
     def parse(self, cr, data):
-        'implements account_banking.parsers.models.parser.parse()'
-        res = super(RaboMT940Parser, self).parse(cr, data + '\n-XXX')
-        return res
-
-    def handle_tag_25(self, cr, data):
-        '''get account owner information'''
-        self.current_statement.local_account = data.split(' ')[0]
-
-    def create_transaction(self, cr):
-        return transaction()
-
-    def handle_tag_60F(self, cr, data):
-        super(RaboMT940Parser, self).handle_tag_60F(cr, data)
-        self.current_statement.id = '%s-%s' % (
-            self.get_unique_account_identifier(
-                cr, self.current_statement.local_account),
-            self.current_statement.id)
-
-    def handle_tag_61(self, cr, data):
-        super(RaboMT940Parser, self).handle_tag_61(cr, data)
+        """Filter Unprintable characters from file data.
+
+        The file contents of the Rabobank tend to contain unprintable
+        characters that prevent proper parsing. These will be removed.
+        """
+        data = ''.join([x for x in data if x in printable])
+        return super(RaboMT940Parser, self).parse(cr, data)
+
+    def handle_tag_60F(self, data):
+        """get start balance and currency"""
+        # For the moment only first 60F record
+        # The alternative would be to split the file and start a new
+        # statement for each 20: tag encountered.
+        stmt = self.current_statement
+        if not stmt.local_currency:
+            stmt.local_currency = data[7:10]
+            stmt.date = datetime.strptime(data[1:7], '%y%m%d')
+            stmt.start_balance = str2amount(data[0], data[10:])
+            stmt.id = '%s-%s' % (
+                self.current_statement.date.strftime('%Y-%m-%d'),
+                self.current_statement.id)
```

Comment:
You are replacing a method to guarantee a unique identifier with something more static. Is this guaranteed to be unique?